lib/strategies/api-strategy.js

```diff
@@ -41,9 +41,10 @@ ApiStrategy.DEFAULT_SCOPE = "appid_default";
 
 /**
  *
- * @param req
- * @param options.scope: The required scopes, seperated by spaces. For example: 'read write update'
- * @param options.audience (optional): the resource URI. If not provided, then options.scope need to be the full (prefixed) scopes.
+ * @param req - an HTTP request object
+ * @param options.scope - The required scopes, separated by spaces. For example: 'read write update'
+ * @param options.audience -  (optional) the resource URI. If not provided, then options.scope need to be the full (prefixed) scopes.
```

Comment:
This senetense need to be removed. change to:
"(optional) the resource URI. it is recommended to provide an audience parameter to guarantee the scopes are for requested app."